src/main/java/uk/gov/hmcts/cpp/audit/bff/client/BaseClient.java

```diff
@@ -0,0 +1,36 @@
+
+package uk.gov.hmcts.cpp.audit.bff.client;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * Base client class providing common HTTP functionality for all API clients.
+ */
+public abstract class BaseClient {
+
+    protected static final String HEADER_USER = "CJSCPPUID";
+    protected static final String HEADER_CORR = "CPPCLIENTCORRELATIONID";
```

Comment:
Constants like these that are referenced in more than once class should be defined once. In the existing services this is done by referencing constants defined [here].(https://github.com/hmcts/cp-framework-libraries/blob/main/framework-api/framework-api-common/src/main/java/uk/gov/justice/services/common/http/HeaderConstants.java)